gtsofa/medium-api
# #166213694 integrate code coverage service to the app

### What is this PR supposed to do?
- Adds an integration code coverage service to the repo
- Add a badge for displaying the coverage
### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out to  this `ch-integrate-code-coverage-service-166213694` branch
- run rake 

### Relevant PT stories
- [#166213694](https://www.pivotaltracker.com/n/projects/2350589/stories/166213694)
